packages/integration-tests/projects/suite-web/plugins/index.js

```diff
@@ -218,5 +224,20 @@ module.exports = on => {
             controller.disconnect();
             return null;
         },
+        readDir: dir => fs.readdirSync(dir, { encoding: 'utf-8' }),
+        rmDir: opts => {
+            const { dir } = opts;
+            let { force, recursive } = opts;
+
+            // just a security check so that we do accidentally wipe something we don't want
+            if (!dir.endsWith('/cypress/downloads')) {
```

Comment:
```suggestion
            if (!dir.endsWith(Cypress.config('downloadsFolder'))) {
```

or what about restricting to delete everything outside `/cypress` scope

Reply:
didn't want to se Cypress.config('downloadsFolder') in case somebody or something misconfigured it.

> or what about restricting to delete everything outside /cypress scope

could do that, need to play a little with path normalization which was something I was kinda lazy to do :]